### ISSUE TITLE
Odd indent error after updating to 1.3.1

### ISSUE THREAD

**Issue by pdehaan** [eslint/eslint#3591]
I'm pretty sure this was working in ESLint v1.2, but I just tried updating a large project to ESLint v1.3.1 and am getting a few odd `indent` rule errors:

### INPUT:
```js
var P = require('bluebird');

var obj = {
  send: function () {
    return P.resolve({
      type: 'POST'
    })
    .then(function () {
      return true;
    }, function () {
      return false;
    });
  }
};

obj.send().then(console.log).catch(console.error);
```

### .eslintrc
```yaml
extend: eslint:recommended

envs:
  node: true

rules:
  indent: [2, 2]
```

### OUTPUT:
```bash
➜  eslint-bluebird  npm test

> eslint-bluebird@1.0.0 test /Users/pdehaan/dev/tmp/del/eslint-bluebird
> eslint --version; eslint .

v1.3.1

index.js
  11:7  error  Expected indentation of 8 space characters but found 6  indent
  12:5  error  Expected indentation of 6 space characters but found 4  indent

✖ 2 problems (2 errors, 0 warnings)

npm ERR! Test failed.  See above for more details.
```

### WORKAROUND:
It seems like ESLint wants me to use this indenting, which looks a bit funky near the end:

```js
var P = require('bluebird');

var obj = {
  send: function () {
    return P.resolve({
      type: 'POST'
    })
    .then(function () {
      return true;
    }, function () {
        return false;
      });
  }
};

obj.send().then(console.log).catch(console.error);
```

**Comment by BYK** [comment]
No idea how this happened but I'm pretty sure it's my fault :) Looking into it.

**Comment by enobufs** [comment]
Has this really fixed? Using the commit 52c42, I am still seeing the similar error with this code:
```js
'use strict';
/*eslint-disable no-console */
var Promise = require('bluebird');

function doSomething() {
    return Promise.resolve()
    .then(function () {
        var o = 1 + 2;
        return o;
    }, function (err) {
        console.log('err:', err);
        var o = 1 - 2;
        return o;
    });
}

doSomething()
.then(function (ans) {
    console.log('ans=%d', ans);
});
```
Here's the output:
```
test.js|11 col 9 error| Expected indentation of 12 space characters but found 8. (indent)
test.js|12 col 9 error| Expected indentation of 12 space characters but found 8. (indent)
test.js|13 col 9 error| Expected indentation of 12 space characters but found 8. (indent)
test.js|14 col 5 error| Expected indentation of 8 space characters but found 4. (indent)
```
My indent rule is `[2, 4]`

**Comment by enobufs** [comment]
Ok, I have confirmed that it fixes the example by @pdehaan, but NOT for the above code.

Should I file a new issue?

**Comment by ilyavolodin** [comment]
Yes, please file a new issue.

**Comment by BYK** [comment]
@enobufs for the future, we do add tests and make sure the sample code reported in the issue is definitely resolved in the patch. See https://github.com/eslint/eslint/pull/3609/files#diff-a28ba1a60512ed1edcdfca32d70ce897R792 for instance.

**Comment by pdehaan** [comment]
I think this is still/differently broken in **eslint@1.4.1**.

```
$ ./node_modules/grunt-eslint/node_modules/.bin/eslint --version
v1.4.1

$ grunt eslint:tests
Running "eslint:tests" (eslint) task

/Users/pdehaan/dev/fxa-content-server_pd/fxa-content-server/tests/functional/oauth_reset_password.js
  40:9  warning  Expected indentation of 6 space characters but found 8  indent

✖ 1 problem (0 errors, 1 warning)


Done, without errors.
```

And here's my code:
```js
37:      return client.signUp(email, PASSWORD, { preVerified: true })
38:        .then(function (result) {
39:          // do nothing
40:        })
41:        .then(function () {
42:          // clear localStorage to avoid polluting other tests.
43:          return FunctionalHelpers.clearBrowserState(self, {
44:            contentServer: true,
45:            '123done': true
46:          });
47:        });
```

And here's the code that ESLint expects:
```js
      return client.signUp(email, PASSWORD, { preVerified: true })
        .then(function (result) {
          // do nothing
      })
        .then(function () {
          // clear localStorage to avoid polluting other tests.
          return FunctionalHelpers.clearBrowserState(self, {
            contentServer: true,
            '123done': true
          });
        });
```

I think this is due to the fact that the only element in my function is a comment.

**Comment by pdehaan** [comment]
Filed new issue as https://github.com/eslint/eslint/issues/3799
